assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/web.xml

```diff
@@ -23,7 +23,7 @@
     </context-param>
     <context-param>
         <param-name>org.eclipse.che.websocket.endpoint</param-name>
-        <param-value>/ws/{ws-id}</param-value>
+        <param-value>/ws</param-value>
```

Comment:
could we have http redirect from /ws/<id> to /ws ?
for backward compliance for existing clients ?